SMRefLocation/schema.json

```diff
@@ -11,21 +11,18 @@
     },
     {
       "properties": {
-	    "type": {
+	  "type": {
           "type": "string",
           "description": "Property. NGSI-LD Entity Type. It must be equal to SMRefLocation.",
           "enum": [
             "SMRefLocation"
           ]
         },
-		"locationName": {
+	  "locationName": {

```

Comment:
use address for this in GSMA-Commons

Reply:
This is just a string, please explain further what can be done as I'm not sure I understand.

Reply:
address is an object, one of its sub-properties is streetAddress, which I guess is what you need.
See this link 
In the part of location-commons
https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons

Reply:
Done, please check schema.json and example.json, they validate correctly.